core/src/com/mygdx/game/assets/GameConfigAssets.java

```diff
@@ -51,27 +54,39 @@ public GameConfigs getGameConfigs() {
     return gameConfigs;
   }
 
+  public void loadAssetsSync() {
+    loadConfigs();
+    assetManager.finishLoading();
+  }
+
   @SuppressWarnings({"unchecked", "rawtypes"})
+  private void initCustomLoaders() {
+    assetManager.setLoader(Array.class, new ArrayLoader(new InternalFileHandleResolver()));
+    for (var aClass : configClasses) {
```

Comment:
aClass trochę słaba nazwa

Reply:
no niby tak, ale w sumie to jest to - mógłbym zmienić na "configClass", ale wtedy może się mylić z "configClasses", a samo "class" to keyword javy